geth/txqueue/txqueue_manager_test.go

```diff
@@ -265,9 +307,11 @@ func (s *TxQueueTestSuite) TestDiscardTransaction() {
 	err := txQueueManager.QueueTransaction(tx)
 	s.NoError(err)
 
+	w := make(chan struct{})
```

Comment:
I am not a fan of one char variables. In such contexts, `done := make(chan struct{})` works pretty well and indicates an intention clearly.